Rakefile

```
@@ -81,7 +81,14 @@ task :new_cop, [:cop] do |_task, args|
   generator.write_source
   generator.write_spec
   generator.inject_require(root_file_path: 'lib/rubocop/cop/rspec_cops.rb')
-  generator.inject_config(config_file_path: 'config/default.yml')
+  generator.inject_config(config_file_path: 'config/default.yml',
+                          version_added: bump_minor_version)
 
   puts generator.todo
 end
+
+def bump_minor_version
+  versions = RuboCop::RSpec::Version::STRING.split('.')
+
+  "#{versions[0]}.#{versions[1].succ}"
```

This will return e.g. `1.39` where I would expect `1.39.0`.

---

We can also be a bit more specific with the local variable names, e.g.
```suggestion
  major, minor, _patch = RuboCop::RSpec::Version::STRING.split('.')

  "#{major}.#{minor.succ}.0"
```